var/spack/repos/builtin/packages/neuron/package.py

```diff
@@ -80,6 +82,9 @@ def cmake_options(spec_options):
         if "+legacy-unit" in spec:
             args.append('-DNRN_DYNAMIC_UNITS_USE_LEGACY=ON')
 
+        if "+caliper" in spec:
+            args.append('-DCORENRN_CALIPER_PROFILEING=ON')
```

Comment:
Is this a typo:
```suggestion
            args.append('-DCORENRN_CALIPER_PROFILING=ON')
```
?

Reply:
of course it is! thank you @alalazo !

Reply:
Done.